react-carousel/src/components/Carousel.js

```diff
@@ -66,7 +66,9 @@ const Carousel = (props) => {
     carouselRef,
   );
 
-  setStrategies(strategies);
+  useEffect(() => {
```

Comment:
👋 

First things first, **thank you for the work you're doing on this**! 🙏 

I was having [the problem that this PR is aiming to fix](https://github.com/brainhubeu/react-carousel/issues/710), so I made a fork to have it available until you guys can merge and release this.

I can confirm that this change works well! 👏 We did see a new warning on the tail of this though. We had an instance of the "Can't perform a React state update on an unmounted component..." error popup on us when we moved between cards in the carousel.

On our fork, we fixed this by checking that the component is still mounted before running `setStrategies`. You can see our implementation at https://github.com/vaam-io/react-carousel/blob/2afb89fc99ef1ddc5ce94b753c412b944d3bcbd1/react-carousel/src/components/Carousel.js#L70.

Let me know if you want me to submit a PR with the changes!

Keep up the good work! 🚀